contribute-to-tidb/write-document.md

```diff
@@ -1,3 +1,229 @@
 # Write Document
 
 Good documentation is crucial for any kind of software. This is especially true for sophisticated software systems such as distributed database like TiDB. The TiDB community aims to provide concise, precise, and complete documentation and welcomes any contribution to improve TiDB's documentation.
+
+## What you can contribute
+
+You can start from any one of the following items to help improve TiDB Docs at the PingCAP website ([en](https://docs.pingcap.com/tidb/stable) and [zh](https://docs.pingcap.com/zh/tidb/stable)):
```

Comment:
It is the truth so I'm ok with it now. But basically we'd better keep neutral on open source software stuff.

cc @winkyao @queenypingcap